Yoast/wordpress-seo
# Moved inline style from toolbar items to stylesheet

## Summary

This PR can be summarized in the following changelog entry:

Improved toolbar item styles for CSP compatibility.
## Relevant technical choices:
- moved inline styles to the toolbar stylesheet
## Test instructions

This PR can be tested by following these steps:

Add the following header code to a test site:

``` php
add_action( 'init', function () {
    header( 'Content-Security-Policy-Report-Only: default-src \'self\'' );
} );
```

Observe no "Refused to apply inline style..." errors in browser console from toolbar items.

Fixes #5607
